integration_tests/commands/http/command_count_test.go

```diff
@@ -0,0 +1,38 @@
+package http
+
+import (
+	"fmt"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandCount(t *testing.T) {
+	exec := NewHTTPCommandExecutor()
+	t.Run("Command count should be positive", func(t *testing.T) {
+		commandCount := getCommandCount(exec)
+		assert.Assert(t, commandCount > 0,
+			fmt.Sprintf("Unexpected number of CLI commands found. expected greater than 0, %f found", commandCount))
+	})
+}
+
+func getCommandCount(exec *HTTPCommandExecutor) float64 {
+	cmd := HTTPCommand{Command: "COMMAND/COUNT", Body: map[string]interface{}{"key": ""}}
```

Comment:
Do we not support commands in form of `COMMAND COUNT`?

Reply:
Looking at the implementation of all `COMMAND` supported, I am following the default implementation. Since the commands are used as path parameter, do you suggest we support something like `/COMMAND%20COUNT` ?